interface/src/lib/stores/socket.ts

```
@@ -47,16 +47,35 @@ function createWebSocket() {
 
 			const binary = payload instanceof ArrayBuffer;
 			listeners.get(binary ? 'binary' : 'message')?.forEach((listener) => listener(payload));
+			
 			try {
 				payload = binary ? msgpack.decode(new Uint8Array(payload)) : JSON.parse(payload);
 			} catch (error) {
+				console.error('[WebSocket] Decode error:', error); // 🌙
 				listeners.get('error')?.forEach((listener) => listener(error));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)